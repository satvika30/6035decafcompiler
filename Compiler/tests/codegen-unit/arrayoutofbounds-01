class Program {
	int a[5];
	
	void main() {
		int i;
		
	    callout("printf","%d\n", a[6]);

	}
}