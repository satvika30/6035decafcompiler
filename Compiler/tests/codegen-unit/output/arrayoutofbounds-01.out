RUNTIME ERROR: Array index out of bounds (6, 38)
